manage.py

```diff
@@ -24,15 +25,15 @@ def create_subparser(
     return subparser
 
 
-def setenv(args: Namespace) -> None:
+def setenv(args: Namespace, *rst: str) -> None:
```

Comment:
`rst` reads like `restructured text`. What about renaming it to `unknown_args` here and below? Also for setenv and make, we should reject unknown arguments.